src/main/java/org/teamresistance/frc/Gyro3D.java

```diff
@@ -0,0 +1,45 @@
+package org.teamresistance.frc;
+
+import com.kauailabs.navx.frc.AHRS;
+import edu.wpi.first.wpilibj.SPI;
+import org.strongback.components.AngleSensor;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Gyro3D implements AngleSensor {
+
```

Comment:
Same nitpick here too: no blank line after the class signature if the next line is an instance variable.